build/AzurePipelinesTemplates/WindowsAppSDK-RunHelixTests-Job.yml

```diff
@@ -241,6 +243,34 @@ jobs:
       PathtoPublish: $(helixTestOutputBaseDir)
       artifactName: HelixTestOutput_$(buildPlatform)_$(buildConfiguration)
 
+  - ${{ if eq(parameters.BuildForHelix, 'true') }}:
+    # Discover the tests to run and generate the Helix Work Items.
+    - task: powershell@2
+      displayName: 'DynamicDependencies.Win32: Discover tests'
+      condition: ${{ parameters.conditionForDiscoverTests }}
+      inputs:
+        targetType: filePath
+        filePath: $(winUIHelixPipelineScripts)\GenerateHelixWorkItems.ps1
+`        arguments: -TestFilePattern 'DynamicDependency_Test_Win32\*.dll' -TestBinaryDirectoryPath '$(testPayloadDir)' -OutputProjFile '$(helixWorkItemsDir)\DynamicDependencies-Tests-Win32.proj' -WorkItemPrefix 'DynamicDependencies.Win32.$(buildPlatform).$(buildConfiguration)' -TestNamePrefix 'DynamicDependencies.Win32.$(buildPlatform).$(buildConfiguration)'
+
+  - ${{ if eq(parameters.BuildForHelix, 'true') }}:
+    # Discover the tests to run and generate the Helix Work Items.
+    - task: powershell@2
+      displayName: 'DynamicDependencies.WinRT: Discover tests'
+      condition: ${{ parameters.conditionForDiscoverTests }}
+      inputs:
+        targetType: filePath
+        filePath: $(winUIHelixPipelineScripts)\GenerateHelixWorkItems.ps1
+`        arguments: -TestFilePattern 'DynamicDependency_Test_WinRT\*.dll' -TestBinaryDirectoryPath '$(testPayloadDir)' -OutputProjFile '$(helixWorkItemsDir)\DynamicDependencies.WinRT-Tests.proj' -WorkItemPrefix 'DynamicDependencies.WinRT.$(buildPlatform).$(buildConfiguration)' -TestNamePrefix 'DynamicDependencies.WinRT.$(buildPlatform).$(buildConfiguration)'
```

Comment:
This should move up above, before the `RunTestsInHelix` happens. (Otherwise it runs too late.)

Probably want to do it near where the MRT core tests call 
`GenerateHelixWorkItems`.

Reply:
How's the update?

Reply:
Looks much better! :)